doc/index.rst

```diff
@@ -32,6 +34,28 @@ Documentation organisation
 Citing Arbor
 ------------
 
+The Arbor software can be cited via Zenodo: |zlatest|
+
+.. |zlatest| image:: https://zenodo.org/badge/DOI/10.5281/zenodo.1459678.svg
+    :target: https://doi.org/10.5281/zenodo.1459678
+
+Previous versions of Arbor can be cited specifically:
+
+* Version 0.5: |z05|
+* Version 0.2: |z02|
+* Version 0.1: |z01|
+
+.. |z05| image:: https://zenodo.org/badge/DOI/10.5281/zenodo.4428108.svg
+    :target: https://doi.org/10.5281/zenodo.4428108
+
+.. |z02| image:: https://zenodo.org/badge/DOI/10.5281/zenodo.2583709.svg
+    :target: https://doi.org/10.5281/zenodo.2583709
+
+.. |z01| image:: https://zenodo.org/badge/DOI/10.5281/zenodo.1459679.svg
+    :target: https://doi.org/10.5281/zenodo.1459679
```

Comment:
Maybe it's worth it to retroactively add Z-IDs for the in-between versions, so that one can cite those as well. That is predicated on this being quick to do, if not, pleasse ignore.

Reply:
Not that quick. It's in the Howto Release on the wiki, so we're not going to miss any future releases.

Reply:
Ok, it's not that important.